lib/src/main/java/com/futurewei/alcor/common/db/ignite/MockIgniteServer.java

```diff
@@ -85,7 +85,14 @@ public static void close() {
 
     public static Ignite getIgnite(){
         // if no need create a real ignite server, we return a mock Ignite client
+        // NOT_JAVA_11_COMPATIBLE: doesn't work in IntelliJ IDE
         return Objects.requireNonNullElseGet(igniteServer, IgniteNodeClientMock::new);
```

Comment:
@pkommoju line 89 should work if you set Java version to 11 in Intellij

Reply:
It gives me runtime error only if I try to debug in the IDE.